images/wkdev_sdk/Containerfile

```diff
@@ -137,5 +137,8 @@ RUN export QT_VERSION=$(qmake6 -query QT_VERSION) && \
       ln -s ${directory} ${directory}/${QT_VERSION}  >/dev/null 2>&1 || true; \
     done
 
+# Check GStreamer plugins are installed.
+RUN gst-inspect-1.0
```

Comment:
One run command, each for the plugin mentioned before :)

RUN gst-inspect-1.0 rsrtp
...